tidy3d/__init__.py

```diff
@@ -256,6 +256,7 @@
 # get material `mat` and variant `var` as `material_library[mat][var]`
 from .material_library.material_library import material_library
 from .material_library.parametric_materials import Graphene
+from .plugins.microwave.rf_material_library import rf_material_library
```

Comment:
no need to expose it to tidy3d namespace. Instead, add it to `plugins/microwave/__init__.py`